src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
Suggestion to clarify? "this node" refers to "CNode" here, which is:

`/** Information about a peer */`

`you` could be rephrased as `your local node`?